docs/_static/css/tethys.css

```diff
@@ -58,3 +58,7 @@ section>section {
     background-color: #cfef90;
 }
 
+pre {
```

Comment:
I didn't like how the theme doesn't have any left/right padding on the code blocks.